docs/content/forge/components/built-in-components.md

```diff
@@ -1,37 +1,54 @@
 # Built-in Components
```

Comment:
Later we should break this down by component

Reply:
What do you mean? Each component has its own header in this file.

Reply:
Each component its own file as we grow the count

Reply:
Not a today problem